app/src/main/java/ar/com/wolox/android/training/ui/training/main/MainFragment.kt

```diff
@@ -0,0 +1,54 @@
+package ar.com.wolox.android.training.ui.training.main
+
+import androidx.core.util.Pair
+import androidx.fragment.app.Fragment
+import ar.com.wolox.android.R
+import ar.com.wolox.android.training.ui.training.main.tabs.news.NewsFragment
+import ar.com.wolox.android.training.ui.training.main.tabs.profile.ProfileFragment
+import ar.com.wolox.wolmo.core.adapter.viewpager.SimpleFragmentPagerAdapter
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import com.google.android.material.tabs.TabLayout
+import kotlinx.android.synthetic.main.fragment_viewpager.*
+import javax.inject.Inject
+
+class MainFragment : WolmoFragment<BasePresenter<Any>>() {
+
+    @Inject internal lateinit var page1Fragment: NewsFragment
+    @Inject internal lateinit var page2Fragment: ProfileFragment
+    private lateinit var fragmentPagerAdapter: SimpleFragmentPagerAdapter
+
+    var tabLayout: TabLayout? = null
```

Comment:
Since tabLayout will be initialized later and then it will be always not null, you could say it's a 
```lateinit var tabLayout: TabLayout```